apps/web_delivery/web/views/error_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule WebDelivery.ErrorView do
+  use WebDelivery.Web, :view
+
+  def render("404.json", _assigns), do: "Not found"
+
+  def render("500.json", _assigns), do: "Internal server error"
```

Comment:
I personally would prefer the safe side, but would it be better to use a sigil instead of escaping the double quotes?